packages/grid/x-grid-data-generator/src/renderer/renderEditContractType.tsx

```diff
@@ -32,9 +32,9 @@ function EditContractType(props: GridCellParams) {
 
   const handleChange = (event) => {
     const editProps = { value: event.target.value };
-    api.setEditCellProps({ id, field, props: editProps }, event);
+    api.setEditCellProps({ id, field, props: editProps });
```

Comment:
In #2220 I changed this component to call the new `setEditCellValue`, which is the correct method and this one does requires the event. We can remove the event now but it will come back later.